build/debian-iptables/Dockerfile

```diff
@@ -14,10 +14,6 @@
 
 FROM BASEIMAGE
```

Comment:
can we still do a follow-up to use ARG instead of sed? :-)

Reply:
yes, but I don't have time to work on that right now.

Reply:
Ok fair enough, I may do this then, taking a note :smile: